stdlib/2and3/time.pyi

```diff
@@ -1,29 +1,26 @@
-# Stubs for time
-# Ron Murawski <ron@horizonchess.com>
-
-# based on: http://docs.python.org/3.3/library/time.html#module-time
-# see: http://nullege.com/codes/search?cq=time
+"""Stub file for the 'time' module."""
+# See https://docs.python.org/3/library/time.html
 
 import sys
-from typing import Any, NamedTuple, Tuple, Union
-from types import SimpleNamespace
+from typing import Any, NamedTuple, Tuple, Union, Optional
+if sys.version_info >= (3, 3):
+    from types import SimpleNamespace
 
 TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
 
-# ----- variables and constants -----
-accept2dyear = False
-altzone = 0
-daylight = 0
-timezone = 0
-tzname = ...  # type: Tuple[str, str]
+accept2dyear: bool = ...
```

Comment:
This doesn't exist in Python 3 (at least not in 3.6 and master, haven't checked all versions).

Reply:
The Python 3.2 documentation marks it as deprecated, Python 3.3 documentation doesn't mention it, so I'll just assume it was removed in 3.3.